<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckForUpdatesDialogClass</class>
 <widget class="QDialog" name="CheckForUpdatesDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Check for updates ...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="messageLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="actionButton">
       <property name="text">
        <string>Action</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CheckForUpdatesDialogClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>552</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionButton</sender>
   <signal>clicked()</signal>
   <receiver>CheckForUpdatesDialogClass</receiver>
   <slot>onProcessAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onProcessAction()</slot>
 </slots>
</ui>
